dummyserver/testcase.py

```diff
@@ -217,3 +218,61 @@ class IPv6HTTPDummyProxyTestCase(HTTPDummyProxyTestCase):
 
     proxy_host = "::1"
     proxy_host_alt = "127.0.0.1"
+
+
+class ConnectionMarker(object):
+    """
+    Marks an HTTP(S)Connection's socket after a request was made.
+    """
+
+    MARK_FORMAT = b"$#MARK%04x*!"
+
+    @classmethod
+    @contextmanager
+    def mark(cls, monkeypatch):
+        """
+        Mark connections under in that context.
+        """
+        from urllib3.connection import HTTPConnection
```

Comment:
Can you please move this import at the top of the file?

Reply:
Done :)